fire/core.py

```diff
@@ -598,6 +611,10 @@ def _CallAndUpdateTrace(component, args, component_trace, treatment='class',
   """
   if not target:
     target = component
+
+  if treatment == 'class':
+    component, treatment, args = _TryClassMethod(component, args)
```

Comment:
Currently you have accessing and calling the class method as a single action.
We should split this into two actions so that the user can access the method without calling it.
This is important so that the user can get help about a class method. Getting help requires accessing the method but not calling it.